RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -179,24 +179,24 @@ class RNUnifiedContacts: NSObject {
     mutableContact.familyName = contactData["familyName"] as! String
     mutableContact.organizationName = contactData["organizationName"] as! String
     
-    for number in contactData["phoneNumbers"] as! NSArray {
-      let transformedNumber = transformNumber( number as! NSDictionary )
-      mutableContact.phoneNumbers.append( transformedNumber )
+    for phoneNumber in contactData["phoneNumbers"] as! NSArray {
+      let phoneNumberAsNSDictionary = convertPhoneNumberToNSDictionary( phoneNumber as! NSDictionary )
```

Comment:
This may have been fault from before. 

This really should be `phoneNumberAsCNLabeledValue` and `convertPhoneNumberToCNLabeledValue`.